viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -65,6 +65,21 @@ static const Eigen::Quaterniond pcdRotation(0.7071068, -0.7071068, 0, 0);
 // This will result in rotations occurring within the y axis to match 2D mapping
 // in the XZ plane
 static const Eigen::Quaterniond pcdOffsetRotation(0.7071068, 0.7071068, 0, 0);
+
+// resolution defines the area in meters that each pixel represent. This
+// is used to draw the jpeg map and in so doing defines the resolution of
+// the outputted PCD
+const double resolution = 0.05;
```

Comment:
I have it set to 0.05 (5 cm) which is different from the original resolution of 0.01 (1 cm) as this produced a 20,000,000 point pointcloud (300 MB) for a run of part of the office. With streaming, we could handle this but the latency is bad as it takes about 30 secs to paint and process. The 0.05 I have currently produces a 12.3MB file with 768,390 points. 

If desired I can do a time analysis and choose a resolution to hit a benchmark

Reply:
Can you let Ray know what the resolution is, and also let him know we can configure this?

Reply:
I spoke with him and he thinks a 5 cm resolution is a good baseline for motion currently but if he needs a different resolution in the interim, he can ping us and we can shoot off a change. He also has been made aware that we will be making this dynamic in the future potentially via the addition of a `GetPointCloudMap` request parameter.

Reply:
great, thank you!

Reply:
would it be worth adding it as a `config_param` in this PR? it would require no changes to rdk/builtin

Reply:
[nit] Change `resolution` to `resolution_meters`

Reply:
Interesting idea John! Eventually, it most likely will be added as a field in the `GetPointCloudMapRequest` message but this could allow us to manipulate it in the interim. Benefits would include being able to control the size of pointclouds being sent over the wire until PCD deltas are available or API changes are made. 

@EshaMaharishi @tessavitabile what do you think?

Reply:
I think it's a good idea to make it configurable in the future, but it's ok to push this PR with it hard-coded as a first step